poetry.lock

```diff
@@ -1,9 +1,10 @@
-# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
```

Comment:
You may want to update poetry, as versions higher than 1.5.x have `category = "dev"` removed.

Reply:
@inaie-makerx This comment hasn't been actioned yet.